packages/udocker/build.sh

```diff
@@ -0,0 +1,12 @@
+TERMUX_PKG_HOMEPAGE=https://indigo-dc.github.io/udocker
+TERMUX_PKG_DESCRIPTION="A basic user tool to execute simple docker containers in batch or interactive systems without root privileges."
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.3.17
+TERMUX_PKG_SRCURL=https://github.com/indigo-dc/udocker/archive/refs/tags/$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=76713c1e8ea3f0f412144fda51b38a6e309d1fe29e85de8f678626d42e9e04a1
+TERMUX_PKG_DEPENDS="python, proot"
+TERMUX_PKG_PYTHON_BUILD_DEPS="wheel"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_PLATFORM_INDEPENDENT=true
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--prefix=$TERMUX_PREFIX"
```

Comment:
`--prefix` is handled automatically by the autotools setup and shouldn't need to be specified separately.
https://github.com/termux/termux-packages/blob/fc2e9c441501d0674b00aa5ea493fc31b64e7de3/scripts/build/configure/termux_step_configure_autotools.sh#L104-L118

Reply:
I used because for some reason it doesn't work in udocker and any libraries go to /usr/local. It's possible it's because it's a python project (setup.py), not an autotools project

Reply:
Ah okay, nevermind then.
That makes sense and might be good to note.